lib/kast.js

```diff
@@ -282,6 +283,8 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "intList" : "",
       ".List{\"intList\"}": "",
       ".WordStack_EVM-DATA": "",
+      "#WordPackUInt112UInt112UInt32(_,_,_)_RULES": "#WordPackUInt112UInt112UInt32",
+      "_[_:=_]_EVM-DATA": "[ := ]",
```

Comment:
Not super happy about this, suggestions welcome.

Reply:
should be infix notated